Java fails to find main class

Commit 21d726f5a204bcaf4b8e1a1430ff3c6f334eb6ab introduced a runtime error:
```
$ git log -n 1 --oneline
21d726f (HEAD -> master, origin/master, origin/HEAD) Bump org.springframework.boot:spring-boot-starter-parent

$ docker build --no-cache -t timveil/cockroachdb-dynamic-certs:latest . && docker run -it timveil/cockroachdb-dynamic-certs:latest
.
.
.
Error: Could not find or load main class org.springframework.boot.loader.JarLauncher
Caused by: java.lang.ClassNotFoundException: org.springframework.boot.loader.JarLauncher
```

Rolling back 1 commit to 1267f6e5d8e34d1128425bc1fbf6bc62a48f5b1c shows no error:
```
$ git log -n 1 --oneline
1267f6e (HEAD) Bump org.springframework.boot:spring-boot-starter-parent

$ docker build --no-cache -t timveil/cockroachdb-dynamic-certs:latest . && docker run -it timveil/cockroachdb-dynamic-certs:latest
.
.
.

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v3.1.5)

2023-11-27T17:28:10.206Z  INFO 1 --- [           main] io.crdb.docker.DynamicCertsApplication   : Starting DynamicCertsApplication v21.0.0-SNAPSHOT using Java 19.0.2 with PID 1 (/BOOT-INF/classes started by root in /)
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -20,4 +20,4 @@ RUN mkdir -pv ./.cockroach-key
 
 EXPOSE 9999
 
-ENTRYPOINT ["java", "org.springframework.boot.loader.JarLauncher"]
+ENTRYPOINT ["java", "org.springframework.boot.loader.launch.JarLauncher"]

